lib/uring/uring_stubs.c

```diff
@@ -419,6 +459,32 @@ ocaml_uring_submit_connect(value v_uring, value v_id, value v_fd, value v_sockad
   CAMLreturn(Val_true);
 }
 
+value
```

Comment:
Needs a comment that v_msghdr must not be GC'd while this is running. Same for `ocaml_uring_submit_recv_msg`.